src/keylessBackup/KeylessBackupProgress.tsx

```diff
@@ -257,15 +270,24 @@ function Restore() {
 
 function Setup({ origin }: { origin: KeylessBackupOrigin }) {
   const keylessBackupStatus = useSelector(keylessBackupStatusSelector)
+  const onboardingProps = useSelector(onboardingPropsSelector)
+  const { step, totalSteps } = getOnboardingStepValues(Screens.SignInWithEmail, onboardingProps)
   const { t } = useTranslation()
+  const { bottom } = useSafeAreaInsets()
+  const insetsStyle = {
+    paddingBottom: Math.max(0, 40 - bottom),
+  }
 
   const navigatedFromSettings = origin === KeylessBackupOrigin.Settings
+  const isOnboarding = origin === KeylessBackupOrigin.Onboarding
```

Comment:
Do we need both `navigatedFromSettings` and `isOnboarding`? One being true should imply that the other is false (unless we have some plan to have a 3rd way to get to CAB).

Reply:
We don't need both, but using `isOnboarding` is more explicit and will keep working if we add another origin.

Reply:
Replaced instances of `navigatedFromSettings` with `isOnboarding` and swapped the tertiaries.